projects/legacy/components/primitive-textfield/primitive-textfield.template.html

```diff
@@ -50,7 +50,7 @@
         </div>
         <div class="t-wrapper">
             <label
-                *ngIf="hasPlaceholder"
+                *ngIf="hasPlaceholder && !isContextTable"
```

Comment:
1) === 'table' can be included in `hasPlaceholder` getter
2) Don't we need placeholder in table when the value is empty? Check out this example:
https://taiga-ui.dev/components/input#table

Reply:
fixed